deploy-odf/verify.sh

```diff
@@ -34,13 +35,25 @@ for edgecluster in ${ALLEDGECLUSTERS}; do
     fi
 
     echo "Check StorageCluster..."
-    if [[ $(oc get --kubeconfig=${EDGE_KUBECONFIG} StorageCluster -n openshift-storage ocs-storagecluster --no-headers | wc -l) -ne 1 ]]; then
-        exit 1
+    if [ "${NUM_M}" -eq "3" ]; then
+        if [[ $(oc get --kubeconfig=${EDGE_KUBECONFIG} StorageCluster -n openshift-storage ocs-storagecluster --no-headers | wc -l) -ne 1 ]] ; then
+            exit 1
+        fi
+    else
+        if [[ $(oc get --kubeconfig=${EDGE_KUBECONFIG} StorageCluster -n openshift-storage mcg-storagecluster  --no-headers | wc -l) -ne 1 ]]; then
+            exit 1
+        fi
     fi
 
     echo "Check StorageClass..."
-    if [[ $(oc get --kubeconfig=${EDGE_KUBECONFIG} sc ocs-storagecluster-ceph-rbd --no-headers | wc -l) -ne 1 ]]; then
-        exit 1
+    if [ "${NUM_M}" -eq "3" ]; then
+        if [[ $(oc get --kubeconfig=${EDGE_KUBECONFIG} sc ocs-storagecluster-ceph-rbd --no-headers | wc -l) -ne 1 ]] ; then
+            exit 1
+        fi
+    else
+        if [[ $(oc get --kubeconfig=${EDGE_KUBECONFIG} sc odf-lvm-vg1 --no-headers | wc -l) -ne 1 ]] ; then
```

Comment:
There should be a storageclass for noobaa. I wonder if it'd be better to check that one instead. Checking on the `lvm` one means that we will make the odf package depend on the `lvm` one, which may change in the future.


![image](https://user-images.githubusercontent.com/13816/185874662-22ad96e4-754b-4aab-93e8-cf5602beb8c0.png)

Reply:
updated.